.vscode/launch.json

```diff
@@ -3,6 +3,12 @@
     "configurations": [
         {
             "type": "node",
+            "request": "attach",
+            "name": "Node Attach",
+            "port": 9229
```

Comment:
Is this port fixed?

Reply:
It is the default port for debugging Node stuff